services/servers/utils/getLatestMemes.js

```diff
@@ -3,15 +3,28 @@ import {
 }  from '../../../constants/index.js';
 import { getMemeResponseEntity } from './index.js';
 
-export const getLatestMemes = async (client, from, to, size) => {
+export const getLatestMemes = async (client, from, to, size, filtersString) => {
```

Comment:
`filtersString` does not follow a particular schema. It's bad to pass arbitrary filters to elastic from the frontend due to the possibility of DOS.

Reply:
I'm not sure that I understand the problem right. Now this is checking for channel and state variable types.